src/ext/oblivc/obliv.h

```diff
@@ -18,6 +18,7 @@ void protocolAddSizeCheck(ProtocolDesc* pd);
 int protocolConnectTcp2P(ProtocolDesc* pd,const char* server,const char* port);
 int protocolAcceptTcp2P(ProtocolDesc* pd,const char* port);
 void cleanupProtocol(ProtocolDesc*);
+void splitProtocol(ProtocolDesc*, ProtocolDesc*);
```

Comment:
See if you can move this into the splitting commit. If not, a separate commit here will also do.